yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -50,8 +50,6 @@ type TableIndexStats struct {
 	ObjectName      string  `json:"ObjectName"`
 	RowCount        *int64  `json:"RowCount"` // Pointer to allows null values
 	ColumnCount     *int64  `json:"ColumnCount"`
-	Reads           *int64  `json:"Reads"`
```

Comment:
What is the reason behind removing this? Is this not used?

Reply:
yes this is not used, confirmed with @shaharuk-yb

Reply:
reads and writes are not used right now. It is safe to remove.